std_msgs/Header header
SemanticObject[] objects
